src/Auth0.OidcClient.Core/IAuth0Client.cs

```diff
@@ -1,66 +1,69 @@
-﻿using System.Collections.Generic;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
 using IdentityModel.OidcClient;
 using IdentityModel.OidcClient.Browser;
 using IdentityModel.OidcClient.Results;
 
 namespace Auth0.OidcClient
 {
+    /// <summary>
+    /// Interface for performing authentication and authorization operations with Auth0 using the
+    /// underlying <see cref="IdentityModel.OidcClient.OidcClient"/>.
+    /// </summary>
     public interface IAuth0Client
     {
         /// <summary>
         /// Launches a browser to log the user in.
         /// </summary>
         /// <param name="extraParameters">Any extra parameters that need to be passed to the authorization endpoint.</param>
-        /// <returns></returns>
+        /// <returns>A <see cref="LoginResult"/> containing the tokens and claims.</returns>
```

Comment:
> containing the tokens and claims

What, exactly, does this provide? If this is an auth code grant, then the auth server response is a code, not tokens. Claims are user attributes within the ID token.

Reply:
It's actually one of these which has tokens and claims https://github.com/IdentityModel/IdentityModel.OidcClient/blob/master/src/Shared/Results/LoginResult.cs